### ISSUE TITLE
/ccenchant minecraft:boots with feather_falling (any value from (1-4) 

### ISSUE THREAD

**Issue by SamCodeArchives** [Earthcomputer/clientcommands#341]
The program cracks RNG and pretty mutch work for me for everything but when it's manupulating RNG for feather falling it does't throw any items

**Comment by Delfite** [comment]
> The program cracks RNG and pretty mutch work for me for everything but when it's manupulating RNG for feather falling it does't throw any items


If you're looking to gets Diamond Boots with these enchantments specifically:
-Depth Strider 3
-Protection 4
-Unbreaking 3
-Feather Falling 4

Then you may need to rejoin your Singleplayer world a few times in order for the RNG to reset properly. I've gotten it to happen previously, I was lucky enough to get in 1 relog before I had gotten the correct seed for it to appear.

If you're doing this on a server that uses the "Paper" server plugin manager then it's impossible to even use /cfish, /ccrackrng,  and /cenchant since paper does some weird backend stuff I won't go into here.

TL;DR: Just relog until you get the correct seed, unless you're on a server that uses paper.

**Comment by zhguyu** [comment]
It's interesting to know that you CAN get Protection 4 and Feather Falling 4 simultaneously, because according to the Wiki the "modified enchantment levels" where those two enchantments can appear do not overlap, which means only  Protection _3_ can be obtained together with Feather Falling 4 in an enchanting table.

**Comment by Earthcomputer** [comment]
Assuming this has been answered
